web/src/beta/components/RadioGroup/RadioBox/index.tsx

```diff
@@ -5,46 +5,45 @@ import { fonts, styled } from "@reearth/services/theme";
 export type Props = {
   inactive?: boolean;
   selected?: boolean;
-  value: string;
+  keyValue: string;
   onClick?: (value: string) => void;
+  label?: string;
 };
 
-const RadioBox: React.FC<Props> = ({ inactive, selected, value, onClick }: Props) => {
-  const [isChecked, setIsChecked] = useState(selected ?? false);
+const RadioBox: React.FC<Props> = ({ selected, keyValue, label, onClick }: Props) => {
+  const [isChecked, setIsChecked] = useState(!!selected);
```

Comment:
Sorry can you update the name here too since we aren't checking it, we are selecting. So `isSelected`

Reply:
done